modules/simple_forms_api/spec/models/vba_40_10007_spec.rb

```diff
@@ -4,32 +4,143 @@
 
 RSpec.describe 'SimpleFormsApi::VBA4010007' do
   describe 'handle_attachments' do
-    it 'saves the combined pdf' do
-      original_pdf = double
-      combined_pdf = double
+    it 'saves the merged pdf' do
+      original_pdf = double('HexaPDF::Document')
+      combined_pdf = double('HexaPDF::Document')
       original_file_path = 'original-file-path'
-      new_file_path = 'new-file-path'
-      form = SimpleFormsApi::VBA400247.new(
+      attachment_page_path = 'attachment_page.pdf'
+      page = double('HexaPDF::Page')
+
+      form = SimpleFormsApi::VBA4010007.new(
```

Comment:
Could this definition be moved to a factory? I think it would greatly help clean up your spec and improve readability at the least

Reply:
Sure

Reply:
@LindseySaari added the factory